### ISSUE TITLE
Crash in BugsnagClient.m line 704

### ISSUE THREAD

**Issue by sethfri** [bugsnag/bugsnag-cocoa#888]
### Describe the bug
Crash of unknown origin coming from Bugsnag. We have seen 13 crashes affecting 13 customers so far.

### Steps to reproduce
Unknown

### Environment
* Bugsnag version: 6.2.3
* CocoaPods version: 1.9.3
* Carthage version (if any): n/a
* iOS/tvOS/macOS version(s): 14.0.1
* Simulator or physical device: iPhone 8 Plus
* XCode version: 12.1
* Swift version (if applicable): 5.1

### Stack Trace

<details><summary>Stack trace:</summary>

```
Crashed: com.apple.main-thread
EXC_BAD_ACCESS KERN_INVALID_ADDRESS 0x000000000000002c
0  MyApp                           0x104bef76c -[BugsnagClient computeDidCrashLastLaunch] + 704 (BugsnagClient.m:704)
1  MyApp                           0x104beee70 -[BugsnagClient start] + 555 (BugsnagClient.m:555)
2  MyApp                           0x104be840c +[Bugsnag startWithConfiguration:] + 93 (Bugsnag.m:93)
3  MyApp                           0x10425cdc0 -[SCDualCrashReporter setUpDualCrashHandler] + 41 (SCDualCrashReporter.m:41)
4  MyApp                           0x10464fa84 AppDelegate.setUp(_:) + 4343003780 (<compiler-generated>:4343003780)
5  MyApp                           0x104653ec4 AppDelegate.application(_:willFinishLaunchingWithOptions:) + 663 (AppDelegate.swift:663)
6  MyApp                           0x104655434 @objc AppDelegate.application(_:willFinishLaunchingWithOptions:) + 4343026740
7  ???                            0x18dc5c4c8 (Missing)
8  ???                            0x18dc5e4f8 (Missing)
9  ???                            0x18dc63cac (Missing)
10 ???                            0x18d342c4c (Missing)
11 ???                            0x18d86582c (Missing)
12 ???                            0x18d343734 (Missing)
13 ???                            0x18d343188 (Missing)
14 ???                            0x18d343560 (Missing)
15 ???                            0x18d342e1c (Missing)
16 ???                            0x18d34a9bc (Missing)
17 ???                            0x18d77f0a0 (Missing)
18 ???                            0x18d87d10c (Missing)
19 ???                            0x18d34a754 (Missing)
20 ???                            0x18d18c9b4 (Missing)
21 ???                            0x18d18b454 (Missing)
22 ???                            0x18d18c600 (Missing)
23 ???                            0x18dc61e80 (Missing)
24 ???                            0x18d7a5914 (Missing)
25 ???                            0x199c9fcb4 (Missing)
26 ???                            0x199cc76e0 (Missing)
27 ???                            0x199cad410 (Missing)
28 ???                            0x199cc73f4 (Missing)
29 ???                            0x18b05a280 (Missing)
30 ???                            0x18b03319c (Missing)
31 ???                            0x199cec4b8 (Missing)
32 ???                            0x199cec180 (Missing)
33 ???                            0x199cec654 (Missing)
34 ???                            0x18b3a1240 (Missing)
35 ???                            0x18b3a1140 (Missing)
36 ???                            0x18b3a0488 (Missing)
37 ???                            0x18b39aa40 (Missing)
38 ???                            0x18b39a200 (Missing)
39 ???                            0x1a1495598 (Missing)
40 ???                            0x18dc60004 (Missing)
41 ???                            0x18dc655d8 (Missing)
42 MyApp                           0x1042a82e0 main + 63 (AppDelegate.swift:63)
43 ???                            0x18b079598 (Missing)
```
</details>

**Comment by johnkiely1** [comment]
Hi @sethfri,

Thanks for raising this, we will take a look at get back to you when we know more.

**Comment by mattdyoung** [comment]
Hi @sethfri 
This looks likely to be a bug so we're investigating further to see if we can reproduce.

**Comment by sethfri** [comment]
Edited the description to indicate that we're actually seeing this on Bugsnag 6.2.1, not 6.2.3. We haven't seen this crash _yet_ with 6.2.3.

**Comment by sethfri** [comment]
We are now seeing this in 6.2.3 as well. Have edited the description again

**Comment by johnkiely1** [comment]
Hi @sethfri, 

We've identified the bug and implemented a fix for this one. It has been released in [v6.3.0](https://github.com/bugsnag/bugsnag-cocoa/releases/tag/v6.3.0)
Thanks for raising the issue.
